zksync_sdk/ethereum_signer/web3.py

```diff
@@ -24,17 +23,18 @@ def sign(self, message: bytes) -> TxEthSignature:
     def address(self) -> str:
         return self.account.address
 
-    def recovery_message(self, tx: TxEthSignature, message: bytes) -> str:
+
+class TxEthValidator(TxEthValidatorInterface):
+
+    def is_valid_signature(self, tx: EncodedTx) -> bool:
```

Comment:
`EncodedTx` already containts signer address. Usually, it is the `from` field. There is no need to make a separate class for that

Reply:
@StanislavBreadless. Current implementation does not have `from` field in all objects including base class. Ok. Let me see what I can do.

Reply:
Also generally saying it needs refactoring because Transfer object, for instance, does not have eth_signature member etc. It would be much easier if it was generalized